### ISSUE TITLE
HMS Games: HuaweiMobileServices.Utils.HMSException

### ISSUE THREAD

**Issue by akirayui** [EvilMindDevs/hms-unity-plugin#97]
Hi, 
I followed the instructions and was able to implement ads successfully with this plugin.
However, when I tested the games demo I encountered an error.
I added sha256 to project on Huawei console.
Also, I created several leaderboards on console.
I tried to sign in with Huawei id, result : SUCCESS
Then I tried to show leaderboards and this exception thrown:  HuaweiMobileServices.Utils.HMSException

Unity version: 2020.1.14
IL2CPP
HMS plugin: v1.2
Huawei P40 Pro - Cloud Debugging.

Log:
`
Unity   : [HMS]: Sign in HuaweiMobileServices.Id.HuaweiIdAuthService
Unity   : [HMS]: Sign in SUCCESS with id: HuaweiMobileServices.Id.AuthHuaweiId
Unity   : [HMS GAMES] ShowLeaderboards ERROR : Exception of type 'HuaweiMobileServices.Utils.HMSException' was thrown.
`

**Comment by smajlo** [comment]
Hello. Have you found a solution?
I have the same problem.
Account sign-in works, But Leaderboards or Achviemenets not working.
I added sha256 to the project setting, also created test users in the console.
I used Unity 2019 ... Mono and IL2CPP .. and different versions of plugins.  Also, I tried the last changes from PR.
Thanks.

Log:
`[HMS GAMES:] ShowAchievements ERROR HuaweiMobileServices.Utils.HMSException: Exception of type 'HuaweiMobileServices.Utils.HMSException' was thrown.`
